Program.cs

```diff
@@ -1,2 +1,66 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿using AirportTicketBookingSystem.Helpers;
+using AirportTicketBookingSystem.Services;
+using AirportTicketBookingSystem.Views;
+using System;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        var bookingService = new BookingService();
+        var flightService = new FlightService();
+        var managerService = new ManagerService();
+        var commandFactory = new CommandFactory(bookingService, flightService, managerService);
```

Comment:
Let's think about how we can do dependency injection here and in every child class.
You'll see the dependency injection pattern ALOT in .NET